hs/Makefile

```diff
@@ -1,6 +1,8 @@
 IMG_HASKELL_BUILD_ARTIFACTS="reachsh/haskell-build-artifacts"
 IMG_REACHC="reachsh/reach"
 IMG_REACH_CLI="reachsh/reach-cli"
+ALPINE_VERSION="3.14.2"
+HOST_IPADDR=$$(../scripts/get-ipaddress.sh)
```

Comment:
can you add this script to the pr?

Reply:
I actually forgot to remove this line, the script won't be necessary anymore. Let me fix this.